pages/docs/reference/properties.md

```diff
@@ -179,13 +179,24 @@ public class MyTest {
 }
 ```
 
-The modifier can only be used on `var` properties declared inside the body of a class (not in the primary constructor), and only
-when the property does not have a custom getter or setter. The type of the property must be non-null, and it must not be
-a primitive type.
+The modifier can be used on `var` properties declared inside the body of a class (not in the primary constructor, and only
+when the property does not have a custom getter or setter) and, since Kotlin 1.2, for top-level properties and 
+local variables. The type of the property or variable must be non-null, and it must not be a primitive type.
 
 Accessing a `lateinit` property before it has been initialized throws a special exception that clearly identifies the property
 being accessed and the fact that it hasn't been initialized.
 
+### Checking whether a lateinit var is initialized (since 1.2)
+
+To check whether a `lateinit var` has already been initialized, use `.isInitialized` on 
```

Comment:
Would be useful to mention that calling `isInitialized` is only possible if the property is accessible lexically (basically, declared in the same class or in one of outer classes).

Reply:
Done.